bobtemplates/plone/addon/buildout.cfg.bob

```diff
@@ -91,6 +91,12 @@ eggs = zest.releaser
 recipe = zc.recipe.egg
 eggs = i18ndude
 
+
+[translate]
+recipe = plone.recipe.command
```

Comment:
We have a problem with this recipe. It only runs when we run buildout. Thats not what we want here. We need to create a script in bin folder. I'll copy my comments from the chat here too, for documentation:

We could use Python entry points for the locales script. We can't use a shell script, because we want also support windows. If we have a simple entry point, we could let buildout generate a script in the bin folder, as we do for the zopepy script for example.